proto/catalog.proto

```diff
@@ -159,6 +159,28 @@ message Sink {
   optional string created_at_cluster_version = 23;
 }
 
+message Subscription {
+  uint32 id = 1;
+  string name = 2;
+  string definition = 3;
+  repeated common.ColumnOrder plan_pk = 5;
+  repeated int32 distribution_key = 6;
+  map<string, string> properties = 7;
+  repeated plan_common.ColumnCatalog column_catalogs = 8;
+  string db_name = 9;
```

Comment:
Are these two fields `db_name` and `subscription_from_name` required? If not I suggest to remove them, otherwise we have to change it accordingly if the target relation name is changed.